common/djangoapps/student/cookies.py

```diff
@@ -58,6 +59,8 @@ def set_logged_in_cookies(request, response, user):
         "username": "test-user",
         "header_urls": {
             "account_settings": "https://example.com/account/settings",
+            "resume_block":
+                "https://example.com//courses/org.0/course_0/Run_0/jump_to/i4x://org.0/course_0/vertical/vertical_4"
```

Comment:
Should we have such a default value for `resume_block`?

Reply:
@schenedx I'm trying to make it so that the dropdown doesn't appear if the field is empty, so no default value.